### PR TITLE
disable multicast discovery by default, log discovery deprecations

### PR DESCRIPTION
**Pull request by kovrus**
https://github.com/crate/elasticsearch/pull/95

### CODE REVIEW

**Comment by mfussenegger** [comment]
"in the next major version" ← is this true?

File: app/src/main/dist/config/crate.yml [hunk 1]
```diff
@@ -484,7 +484,7 @@
 # Therefore, you need to specify the instances you want to connect to a
 # cluster as described in the following steps:
 #
-# 1. Disable multicast discovery (enabled by default):
```
Comment (mfussenegger) [inline app/src/main/dist/config/crate.yml]: I'd rephrase it to Enable multicast discovery and change the value below to true

**Comment by kovrus** [comment]
@mfussenegger https://github.com/crate/crate/blob/master/CHANGES.txt#L27 Naomi has added the cahngelog entry for the multicast deprecation.

Regarding "in the next major version". 1.1 seems to be a major? If it is not true, then i need to patch the upstream..

**Comment by kovrus** [comment]
@mfussenegger fixup
